app/main.py

```diff
@@ -1,2 +1,58 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.size = 0
+        self.threshold = int(self.capacity * 2 / 3)
+        self.hash_table = self.make_hash_table()
+
+    def make_hash_table(self) -> list:
+        return [[] for _ in range(self.capacity)]
+
+    def __len__(self) -> int:
+        return self.size
+
+    def __getitem__(self, key: Hashable) -> Any:
```

Comment:
How does this method behave when the passed key is not in dictionary and there's no collision? Is this a constant time operation in this case?

Reply:
Ah, yes, it seems that it is. It's just achieved in a way that is unexpected for me.